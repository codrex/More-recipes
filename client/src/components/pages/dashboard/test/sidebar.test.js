import React from 'react';
import Sidebar from '../sidebar';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

describe('Sidebar component ', () => {
  test('render as expected without any recipe', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Sidebar user={{}} />
      </BrowserRouter>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
