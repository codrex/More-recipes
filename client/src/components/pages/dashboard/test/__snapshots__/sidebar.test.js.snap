// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component  render as expected without any recipe 1`] = `
<div
  className="sidebar col-12 "
>
  <div
    className="sidebar-header"
  >
    <div
      className="avatar avatar-sm"
    >
      
    </div>
    <h4
      className="lead sidebar-text"
    >
      Dashboard
    </h4>
    <a
      href="/user"
      onClick={[Function]}
    >
      <p
        className="small text-white text-capitalize"
      >
        my profile
      </p>
    </a>
  </div>
  <div
    className="sidebar-body"
  >
    <h4
      className="lead sidebar-text sidebar-list-item active"
      onClick={[Function]}
    >
      All recipes
    </h4>
    <h4
      className="lead sidebar-text sidebar-list-item"
      onClick={[Function]}
    >
      Top recipes
    </h4>
    <h4
      className="lead sidebar-text sidebar-list-item"
      onClick={[Function]}
    >
      Favorite recipes
    </h4>
  </div>
</div>
`;
